src/components/App.tsx

```diff
@@ -551,6 +565,7 @@ export class App extends React.Component<AppProps, AppState> {
             group.open(file);
             self.setState({ group });
           }}
+          onPreview={self.showPreview.bind(self)}
```

Comment:
use arrow function instead of bind `onPreview={() => self.showPreview()}

Reply:
Or ever better rename showPreview to onPreview in class body with arrow function definition in this manner:

```js
class App extends React.Component {
    onPreview = () => {
       if (file.type === FileType.Markdown) {
           ...
       }
    }
     render() {
         return (
             <EditorPane
                 onPreview={this.onPreview}
            />;
        );
    }
}